### PR TITLE
bugfixs :  pipeleine_option has attribute 'validate' may cause exception

### PR DESCRIPTION
**Pull request by YuanGunGun**
If a PipelineOptions implement has attribute named ‘validate’ , it may cause exception

### CODE REVIEW

File: sdks/python/apache_beam/options/pipeline_options_validator.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def validate(self):
     """
     errors = []
     for cls in self.OPTIONS:
-      if 'validate' in cls.__dict__:
+      if 'validate' in [k for k, v in cls.__dict__.items() if str(v).startswith("<function")]:
```
Comment (pabloem) [inline sdks/python/apache_beam/options/pipeline_options_validator.py]: Can you just do: `if 'validate' in cls.__dict__ and callable(cls.__dict__['validate'])`?

Also, please make sure that the line is less than 80-char long : )

Comment (YuanGunGun) [reply sdks/python/apache_beam/options/pipeline_options_validator.py]: got it, i will change

**Comment by YuanGunGun** [comment]
@pabloem please check again

**Comment by YuanGunGun** [comment]
@pabloem change it

**Comment by YuanGunGun** [comment]
@pabloem Very confused not merge , anything else need I do ?

**Comment by pabloem** [comment]
Hi ! Sorry that I missed this. I wasn't able to track it properly.

**Pull request merged**